middle_end/flambda2/terms/cost_metrics.ml

```diff
@@ -70,13 +70,18 @@ let set_of_closures ~find_code_characteristics set_of_closures =
   in
   let cost_metrics, num_words =
     Function_slot.Map.fold
-      (fun _ code_id (metrics, num_words) ->
-        let { cost_metrics; params_arity } =
-          find_code_characteristics code_id
-        in
-        ( metrics + cost_metrics,
-          (* CR poechsel: valid until OCaml 4.12, as for named_size *)
-          Stdlib.( + ) num_words (if params_arity <= 1 then 2 else 3) ))
+      (fun _ (code_id : Function_declarations.code_id_in_function_declaration)
+           (metrics, num_words) ->
+        match code_id with
+        | Deleted { function_slot_size } ->
+          metrics, Stdlib.( + ) num_words function_slot_size (* XXX *)
```

Comment:
Why the `(* XXX *)` comment ? This looks as correct as the code below (i.e. slightly wrong because it ignores the infix tag).